staging/src/k8s.io/client-go/util/workqueue/parallelizer.go

```diff
@@ -25,6 +25,9 @@ import (
 
 type DoWorkPieceFunc func(piece int)
 
+// DEPRECATED: Parallelize is deprecated. ParallelizeUntil is a more
```

Comment:
this should be in the default deprecation style, compare https://rakyll.org/deprecated/

Reply:
DONE. Thanks.